diffblue-base.yml

```diff
@@ -1,13 +0,0 @@
-# This file contains configuration for Diffblue Cover that is specific for
-# this project, typically a custom buildCmd, classpathCmd and
-# java-version (as a CBMC argument).
-#
-# Other general options such as skipVerification, ignoreExistingCoverage, CBMC
-# arguments (except for java-version) and phases SHOULD NOT be specified here.
-# They are part of diffblue.yml file that is automatically generated by
-# merging a custom yml file on top of this one using the script
-# in dashboard/scripts/update-diffblue-yml. Any options specified here would
-# be overriden to keep consistency with other benchmarks.
-# mvn -X adds debugging info to the build output which is necessary to build
-# the classpath.
-buildCmd: cd src/supernode; mvn -e -X -Dmaven.test.skip=true compile
```

Comment:
How is this now taken care of now? It's true this file is no longer used but this command does not appear in the `diffblue-base.json`, please make sure this has been taken care of. @JohnLBergqvist Can you perhaps clarify this?

Reply:
the location of the source code for the module (`src/supernode`) is specified in firebase: https://console.firebase.google.com/project/diffblue-dashboard/database/diffblue-dashboard/data/benchmarks/Dragonfly/modules

Reply:
I asked him before. We just upload the jars now, so we don't need to run maven.